### ISSUE TITLE
Can't select databases to load by their level

### ISSUE THREAD

**Issue by ygoe** [extremeshok/clamav-unofficial-sigs#323]
I'm not sure where to write this because this repository is the only place accessible about these databases and the script. The linked website is empty (error 404).

I've installed the script on an Ubuntu system and it has loaded certain databases on its own. From the list of loaded databases, as explained in the readme file, I see that LOW and MEDIUM databases are loaded. I want to change that to LOW only but can't find out how to do that. There's nothing in the three config files or this repository readme. Is that documentation still missing? Or it this not possible without editing the list of databases in master.conf? But I'm wondering why HIGH databases are not loaded, despite being included in master.conf and not commented out.

**Comment by ygoe** [comment]
After reading parts of the script, I learned that the `default_dbs_rating` setting is the filter preference. Set this to "LOW" to only load databases with a LOW rating. It's undocumented so you really have to inspect the source code to find this setting. But it isn't as effective as I expected it (again, nothing is documented).

It seems databases that were once activated are never deleted again. Running the script and restarting services doesn't help at all. I resolved this by completely uninstalling this script and clamav including all configuration files, and installing it all again. But this time I had to edit the user.conf file before anything was downloaded. Now only LOW databases are active.

Is there a better way to apply configuration changes than deleting everything and starting all over with the new config?
